diff --git a/web/.htaccess b/web/.htaccess
index 4031da47..5a9c01a0 100644
--- a/web/.htaccess
+++ b/web/.htaccess
@@ -113,6 +113,11 @@ AddEncoding gzip svgz
   # uncomment the following line:
   # RewriteBase /
 
+  # Allow direct access to the /slides directory
+  RewriteCond %{REQUEST_URI} ^/slides/ [NC]
+  RewriteCond %{DOCUMENT_ROOT}/web/slides -d
+  RewriteRule ^slides/(.*)$ /web/slides/$1 [L]
+
   # Redirect common PHP files to their new locations.
   RewriteCond %{REQUEST_URI} ^(.*)?/(install\.php) [OR]
   RewriteCond %{REQUEST_URI} ^(.*)?/(rebuild\.php)

